README.md

```diff
@@ -25,20 +25,35 @@ pip install synapse-token-authenticator
 ## Configuration
 Here are the available configuration options:
 ```yaml
-# provide only one of secret, keyfile
-secret: symetrical secret
-keyfile: path to asymetrical keyfile
-
-# Algorithm of the tokens, defaults to HS512
-#algorithm: HS512
-# Allow registration of new users using these tokens, defaults to false
-#allow_registration: false
-# Require tokens to have an expiracy set, defaults to true
-#require_expiracy: true
+jwt:
+  # provide only one of secret, keyfile
+  secret: symetrical secret
+  keyfile: path to asymetrical keyfile
+
+  # Algorithm of the tokens, defaults to HS512 (optional)
+  algorithm: HS512
+  # Allow registration of new users, defaults to false (optional)
+  allow_registration: false
+  # Require tokens to have an expiry set, defaults to true (optional)
+  require_expiry: true
+oidc:
+  issuer: "https://idp.example.com"
+  client_id: "<IDP client id>"
+  client_secret: "<IDP client secret>"
+  # Zitadel Organization ID, used for masking. (Optional)
+  organization_id: 1234
+  # Zitadel Project ID, used for validating the audience of the returned token.
+  project_id: 5678
+  # Limits access to specified clients. Allows any client if not set (optional)
+  allowed_client_ids: ['2897827328738@project_name']
+  # Allow registration of new users, defaults to false (optional)
+  allow_registration: false
 ```
```

Comment:
We could add the correct config here:

```yaml
password_providers:
    - module: "synapse_token_authenticator.TokenAuthenticator"
    config:
      jwt:
        # provide only one of secret, keyfile
        secret: symetrical secret
        keyfile: path to asymetrical keyfile
      
        # Algorithm of the tokens, defaults to HS512 (optional)
        algorithm: HS512
        # Allow registration of new users, defaults to false (optional)
        allow_registration: false
        # Require tokens to have an expiry set, defaults to true (optional)
        require_expiry: true
      oidc:
        issuer: "https://idp.example.com"
        client_id: "<IDP client id>"
        client_secret: "<IDP client secret>"
        # Zitadel Organization ID, used for masking. (Optional)
        organization_id: 1234
        # Zitadel Project ID, used for validating the audience of the returned token.
        project_id: 5678
        # Limits access to specified clients. Allows any client if not set (optional)
        allowed_client_ids: ['2897827328738@project_name']
        # Allow registration of new users, defaults to false (optional)
        allow_registration: false
```

Reply:
I would either propose merging this PR and updating #27, or getting #27 merged and then updating this PR

Reply:
As you like